surveys/2022-annual-survey/questions.md

```diff
@@ -26,12 +26,65 @@ Type: select one
 
 ## For previous Rust users
 
+### Why did you stop using Rust?
+
+Type: select all that apply
+
+- Rust did not help me achieve my goals
+- Missing language features
+- Missing libraries
+- Missing tools
+- Too difficult to learn
+- Community was rude, unwelcoming, or otherwise off-putting
+- I prefer to use another language
+- I no longer have the opportunity to use Rust due to external factors
+- Other
+
+### Tell us more:
+
+Type: free text
+
+### Would you be likely to use Rust if the above issues were addressed
```

Comment:
I'm not quite sure what we hope to learn from this? Are we suspecting that some users will answer the question in some way that we can ignore because they will also say that they won't use Rust even if that issue is ignored?